tests/test_miscellaneous.py

```diff
@@ -75,6 +75,34 @@ def check_parent(child, parent):
     def test_str(self):
         assert_equal(str(Granularity.SOFTWARE), 'SOFTWARE')
 
+    def test_comparisons(self):
+        assert_equal(Granularity.SOFTWARE, Granularity.SOFTWARE)
+        assert_equal(Granularity.PACKAGE, Granularity.PACKAGE)
+        assert_equal(Granularity.CLASS, Granularity.CLASS)
+        assert_equal(Granularity.METHOD, Granularity.METHOD)
+        assert_equal(Granularity.FUNCTION, Granularity.FUNCTION)
+
+        assert_true(Granularity.SOFTWARE > Granularity.PACKAGE)
```

Comment:
Using generators now. But I think it seems better to have the comparison symbols explicitly written, for we are overwriting those methods.